bundles/org.openhab.binding.dominoswiss/README.md

```diff
@@ -0,0 +1,110 @@
+# Dominoswiss Binding
+
+This binding allows the controll of rollershutters, using an egate as gateway and dominoswiss radio receivers. The egate-gateway is connectet via ethernet to openhab and sends it's commands via radio to all rollershutters. See https://www.brelag.com/ for more information.
+
+
+## Supported Things
+
+EGate: Dominoswiss EGate Server. The egate is the gateway which sends the commands to the connected rollershutters. 
+Blind: represents one rollershutter, that can be controlled via EGate
+
+## Discovery
+
+Unfortunately no automatic discovery is possible due to protocol restrictions. Every rollershutter must be known by egate and can be called by it's number of storage-place on egate gateway.
+
+
+## Binding Configuration
+
+_If your binding requires or supports general configuration settings, please create a folder ```cfg``` and place the configuration file ```<bindingId>.cfg``` inside it. In this section, you should link to this file and provide some information about the options. The file could e.g. look like:_
```

Comment:
Does this apply to your binding? If it does, why not use an example for your binding and get rid of Hue example?